### PR TITLE
First page

### PR DESCRIPTION
**Pull request by FreytagDom**

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ to {
+   -webkit-filter: hue-rotate(-360deg);
+ }
+`;
+
+const Headline = styled.h1`
+  background-image: -webkit-linear-gradient(92deg, #f35626, #feab3a);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  -webkit-animation: ${hue} 10s infinite linear;
```
Comment (ThomasKirst) [inline components/Header.js:25]: 🥳

File: components/HomeInput.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+  return (
+    <>
+      <EntryForm onSubmit={handleSubmit}>
+        <LabelBz htmlFor="bl">
+          Blutzuckerwert <br />
+          mg/dl
+          <DataInput
+            type="number"
+            name="bl"
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
            name="bloodsugar"
```

File: components/HomeInput.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    event.preventDefault();
+
+    const form = event.target;
+    const bl = form.bl.value;
+    const basal = form.basal.value;
+    const bolus = form.bolus.value;
+
+    const cardData = {
+      id: nanoid(),
+      bl: bl,
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
      bloodsugar: bloodsugar,
```

File: components/HomeInput.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  return (
+    <>
+      <EntryForm onSubmit={handleSubmit}>
+        <LabelBz htmlFor="bl">
+          Blutzuckerwert <br />
+          mg/dl
+          <DataInput
+            type="number"
+            name="bl"
+            placeholder={`letzter Blutzuckerwert `}
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
            placeholder="letzter Blutzuckerwert"
```

File: components/HomeInput.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            required
+          />
+        </LabelBz>
+
+        <LabelIu htmlFor="basal">
+          Basal Insulin <br /> Einheiten
+          <DataInput
+            type="decimal"
+            name="basal"
+            placeholder={`letzte Einheit 2,5 `}
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
            placeholder="letzte Einheit 2,5"
```

File: components/HomeInput.js [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+            id="basal"
+          />
+        </LabelIu>
+
+        <LabelIu htmlFor="bolus">
+          Bolus Insulin <br /> Einheiten
+          <DataInput
+            type="decimal"
+            name="bolus"
+            placeholder={`letzte Einheit 2,5`}
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
            placeholder="letzte Einheit 2,5"
```

File: components/HomeInput.js [hunk 7]
```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components';
+import { nanoid } from 'nanoid';
+
+export default function Input({ onAddData }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const form = event.target;
+    const bl = form.bl.value;
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
    const bloodsugar = form.bloodsuger.value;
```

File: components/HomeInput.js [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+`;
+
+const LabelIu = styled.label`
+  color: #5c940d;
+  background-color: beige;
+  border-radius: 8px;
+  display: grid;
+  text-align: center;
+  height: 60px;
+`;
```
Comment (ThomasKirst) [inline components/HomeInput.js]:
```suggestion
const Label = styled.label`
  background-color: beige;
  border-radius: 8px;
  display: grid;
  text-align: center;
  height: 60px;
`
const LabelBz = styled(Label)`
  color: #c92a2a;
`;

const LabelIu = styled(Label)`
  color: #5c940d;
```

File: pages/index.js [hunk 9]
```diff
@@ -1,3 +1,69 @@
...
+          return (
+            <Saved key={item.id}>
+              <BloodSugar> Blutzuckerwert: {item.bl} mg/dl </BloodSugar>
+              <br />
+              <Insulin> Basal Insulin: {item.basal} Einheiten </Insulin>
+              <br />
+              <Insulin> Bolus Insulin: {item.bolus} Einheiten </Insulin>
+            </Saved>
+          );
+        })}
```
Comment (ThomasKirst) [inline pages/index.js]:
```suggestion
        {data.map((item) => 
            <Saved key={item.id}>
              <BloodSugar> Blutzuckerwert: {item.bl} mg/dl </BloodSugar>
              <br />
              <Insulin> Basal Insulin: {item.basal} Einheiten </Insulin>
              <br />
              <Insulin> Bolus Insulin: {item.bolus} Einheiten </Insulin>
            </Saved>
          );
        )}
```

File: pages/index.js [hunk 10]
```diff
@@ -1,3 +1,69 @@
...
+  text-align: center;
+  height: 100px;
+  align-items: center;
+  margin: 20px;
+`;
+
+const BloodSugar = styled.span`
+  color: #c92a2a;
+  margin: 0;
+  padding: 0;
```
Comment (ThomasKirst) [inline pages/index.js:62]: Are they really needed?

File: pages/index.js [hunk 11]
```diff
@@ -1,3 +1,69 @@
...
+`;
+
+const Insulin = styled.span`
+  color: #5c940d;
+  margin: 0;
+  padding: 0;
```
Comment (ThomasKirst) [inline pages/index.js:68]: Same same?

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Looks great so far, I found only a few issues.
You can remove the `styles` folder since there are files which are not used.

**Comment by FreytagDom** [comment]
Thanks for the comments, I have implemented the changes.

**Review by ThomasKirst — APPROVED** [review]
Good job 👍

**Pull request merged**